mailscanner/style.css

```diff
@@ -999,3 +999,6 @@ table.mail>tbody>tr>th.alignleft {
     text-align: left;
 }
 
+a {
```

Comment:
Can you limit this css rule to that detail.php specific case?

Reply:
Hello @endelwar!
A way to to this?

Reply:
You can add a class to the `<a href="..." class="whatever"` and apply it only to that class

Reply:
Done.